guides/source/active_record_migrations.md

```diff
@@ -589,14 +718,15 @@ add_foreign_key :articles, :authors
 ```
 
 This [`add_foreign_key`][] call adds a new constraint to the `articles` table.
-The constraint guarantees that a row in the `authors` table exists where
-the `id` column matches the `articles.author_id`.
+The constraint guarantees that a row in the `authors` table exists where the
+`id` column matches the `articles.author_id`.
 
-If the `from_table` column name cannot be derived from the `to_table` name,
-you can use the `:column` option. Use the `:primary_key` option if the
-referenced primary key is not `:id`.
+If the `from_table` column name cannot be derived from the `to_table` name, you
```

Comment:
"If the `from_table` column name cannot be derived from the `to_table` name" - what does this mean? Can this be made more clear?

Reply:
This was just quoted from the API guides, I've expanded the definition in the description and avoided using the param names `from_table` and `to_table`. Hopefully this, together with the example below helps explain it better. 

```
If the column name of the table to which we're adding the foreign key cannot be
derived from the table with the referenced primary key then you can use the
`:column` option to specify the column name. Additionally, you can use the
`:primary_key` option if the referenced primary key is not `:id`.
```